Lib/multiprocessing/managers.py

```diff
@@ -312,7 +315,7 @@ def fallback_repr(self, conn, ident, obj):
     def dummy(self, c):
         pass
 
-    def debug_info(self, c):
+    def debug_info(self, c): # c is unused?
```

Comment:
Yes. I was wondering whether it could be used for additional debugging information. Admittedly, anything that called it should have the connection anyway!